RoadMap_Movie/RoadMap_Movie/Service/Helpers/KeyChainService.swift

```diff
@@ -0,0 +1,26 @@
+// KeyChainService.swift
+// Copyright © RoadMap. All rights reserved.
+
+import KeychainSwift
+
+/// Сервис для сохранения критических данных
+final class KeyChainService {
+    static let shared = KeyChainService()
+    private init() {}
+
+    // MARK: - Public methods
+
+    func getAPIKey() -> String {
+        guard let apiKey = KeychainSwift().get(Constants.keyChainKey) else {
+            saveAPIKey()
+            return getAPIKey()
+        }
+        return apiKey
+    }
+
+    // MARK: - Private methods
+
+    private func saveAPIKey() {
+        KeychainSwift().set(RequestParameters.apiKeyValue, forKey: Constants.keyChainKey)
```

Comment:
```suggestion
        KeychainSwift().set(, forKey: Constants.keyChainKey)
```
в явно виде ключе не держим в приле